numpyro/distributions/util.py

```diff
@@ -395,6 +282,103 @@ def multinomial_rvs(key, n, p, shape=()):
     return np.reshape(samples_2D, shape + p.shape[-1:]) - excess
 
 
+def multinomial(key, p, n, shape=()):
+    return _multinomial(key, p, n, shape)
+
+
+def _xlogy_jvp_lhs(g, x, y):
+    shape = lax.broadcast_shapes(np.shape(g), np.shape(y))
+    g = np.broadcast_to(g, shape)
+    y = np.broadcast_to(y, shape)
```

Comment:
`lax._brcast` is not exposed (changed to `lax.lax._brcast`) in the newer version of JAX, so I use `np.broadcast_to` instead